DTO/ImportLogDTO.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+declare(strict_types=1);
+
+namespace MauticPlugin\CustomObjectsBundle\DTO;
+
+class ImportLogDTO
+{
+    private array $warnings = [];
+
+    public function hasWarning(): bool
+    {
+        return !empty($this->warnings);
+    }
+
+    public function addWarning(string $warning): void
+    {
+        $this->warnings[] = $warning;
+    }
+
+    public function getFormattedWarning(): string
```

Comment:
```suggestion
    public function getWarningsAsString(): string
```
`getWarningsAsString()` might be a better name.

Reply:
@fedys Removed `getFormattedWarning()` and added `getWarnings()` method as we need warning array for ImportProcessEvent;